recipe/bld.bat

```diff
@@ -0,0 +1,19 @@
+set LIBRARY_PREFIX=%LIBRARY_PREFIX:\=/%
+set MSYSTEM=MINGW%ARCH%
+set MSYS2_PATH_TYPE=inherit
+set CHERE_INVOKING=1
```

Comment:
These seem to be a good addition for that conda-forge environment package which should be installed on all CI services prior to the build...?